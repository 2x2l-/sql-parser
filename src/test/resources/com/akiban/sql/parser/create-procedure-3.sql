CREATE PROCEDURE test.foo_too(IN x INT, OUT y DOUBLE) LANGUAGE JAVASCRIPT AS 'y = x;'