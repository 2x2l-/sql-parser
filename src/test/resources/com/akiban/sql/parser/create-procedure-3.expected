com.akiban.sql.parser.CreateAliasNode@258b678c
aliasType: PROCEDURE
aliasInfo: (IN x INTEGER, OUT y DOUBLE) LANGUAGE javascript
definition: y = x;
name: test.foo_too
statementType: CREATE PROCEDURE